src/components/SignupPage.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+class SignupPage extends React.Component {
```

Comment:
Almost there, just missing the verification of a valid email and password and changing the colors.